pkg/kubelet/cm/container_manager.go

```diff
@@ -60,6 +60,9 @@ type ContainerManager interface {
 	// GetNodeAllocatable returns the amount of compute resources that have to be reserved from scheduling.
 	GetNodeAllocatableReservation() v1.ResourceList
 
+	// GetNodeCapacity returns the amount of compute resources tracked by container manager available on the node.
+	GetCapacity() v1.ResourceList
+
```

Comment:
I tried removing the function from cadvisor.Interface, but it looks like it is required in the eviction.DiskInfoProvider interface.